css/style.css

```
@@ -16,4 +24,101 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+#nav-bar {position:fixed;
+          top:0; 
+          height: 70px;
```

```suggestion
          height: 4.5rem;
```
We should use relative units where possible. Same applies to all the other uses of `px` in this file